conda-recipe/meta.yaml

```diff
@@ -27,6 +27,8 @@ requirements:
   - coloredlogs
   - entrypoints
   - jinja2
+  - pcdsutils
```

Comment:
It's not obvious where to put these without this extra info:
* The `requirements` section here is largely equivalent to `requirements.txt`(i.e., what you need to run the package in general)
* The `test` / `requires` section below is roughly equivalent to `dev-requirements.txt` (i.e., what you need to run the package test suite *in addition to* the run requirements above)
* `docs-requirements.txt` doesn't have an equivalent in the conda recipe, because we build our documentation in a pip-based job

That is to say, these 2 packages should go down to the `dev-requirements` equivalent section (test/requires below)

Reply:
Oh okay. Makes sense, will fix.